packages/lesswrong/components/posts/TableOfContents/TableOfContentsRow.tsx

```diff
@@ -48,7 +48,7 @@ const styles = (theme: ThemeType) => ({
     color: theme.palette.link.tocLink,
     lineHeight: fullHeightToCEnabled ? "1em" : "1.2em",
     '&:hover':{
```

Comment:
I'm not sure if this opacity-change was important for EA Forum. @s-cheng

Reply:
Yeah we do want links to darken on hover - can you keep that style for FriendlyUI?